src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -506,6 +540,17 @@
         )
     }
```

Comment:
here actually we should leave sastOnServerEnabled as null in case of service<SnykApiService>().sastOnServerEnabled return null
we should do checks later in the code (as we do now in Snyk Code related scan)